src/moduler/aktivitet/aktivitet-kort/AktivitetskortTillegg.tsx

```diff
@@ -9,25 +9,34 @@ import AvtaltMarkering from '../avtalt-markering/avtalt-markering';
 import SokeStatusEtikett from '../etikett/SokeStatusEtikett';
 import DialogIkon from '../visning/underelement-for-aktivitet/dialog/DialogIkon';
 import styles from './Aktivitetskort.module.less';
+import IkkeDeltMarkering from "../ikke-delt-markering/IkkeDeltMarkering";
+import {MOTE_TYPE, SAMTALEREFERAT_TYPE} from "../../../constant";
+import {selectErVeileder} from "../../identitet/identitet-selector";
 
 interface Props {
     aktivitet: Aktivitet;
 }
 
-function AktivitetskortTillegg({ aktivitet }: Props) {
+const AktivitetskortTillegg = ({ aktivitet }: Props) => {
     const { avtalt, id, etikett } = aktivitet;
     const dialog = useSelector((state) => selectDialogForAktivitetId(state, id), shallowEqual);
     const henvendelser = dialog ? dialog.henvendelser : [];
     const ulesteHenvendelser = henvendelser.filter((h: Henvendelse) => !h.lest).length;
 
-    if (!(avtalt || !!etikett || !!dialog)) {
+    const erVeileder = useSelector((state) =>  selectErVeileder(state));
```

Comment:
Dette kan også skrives sånn her 😊 

```suggestion
    const erVeileder = useSelector(selectErVeileder);
```